.eslintrc.json

```diff
@@ -14,7 +14,8 @@
   "globals": {
     "flowchart": true, // Blueprints support lib
     "__": true, // Zanata parsing cue
-    "N_": true // Zanata parsing cue
+    "N_": true, // Zanata parsing cue
+    "$window": true
```

Comment:
this shouldn't be necessary, $window is an angular service, if properly injected it'll be defined, cc wont complain